tests/unit/test_auto_regression.py

```diff
@@ -456,14 +456,16 @@ def test_fit_auto_regression_xr_1D_values():
     # statsmodels.tsa.ar_model.AutoReg
 
     data = trend_data_1D()
-    result = mesmer.stats.fit_auto_regression(data, "time", lags=1)
+    lags = 1
+    result = mesmer.stats.fit_auto_regression(data, "time", lags=lags)
 
     expected = xr.Dataset(
         {
             "intercept": 1.04728995,
             "coeffs": ("lags", [0.99682459]),
             "variance": 1.05381192,
             "lags": [1],
+            "nobs": len(data.values) - lags,
```

Comment:
```suggestion
            "nobs": data["time"].size - lags,
```

`len` works but I don't use it for arrays - I prefer `arr.shape[axis]` (or `ds.sizes["time"]` for xarray objects).

(or `arr.size` if it's 1D (the total number of elements on the array))

Reply:
> What's the forgotten entry?
In _fit_auto_regression_np the documentation still said it returns std instead of variance, I adapted the description.

> So yes we'll have to eventually do this but maybe not here.
Ok.